### PR TITLE
Milestone: Add sections

### PR DESCRIPTION
**Pull request by ForHemer**
The goal of this milestone is to add several sections to my portfolio website. This feature, in addition to  the home section, contains others like:

- **About myself:** A section on who am I?, my skills, etc.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,32 +2,174 @@
...
+                        <li><a href="http://twitter.com/HemersonForeste"><img src="images/bird.svg" alt="twitter"></a></li>
+                        <li><a href="https://www.linkedin.com/in/hemerson-foreste-890685197/"><img src="images/Linkedin.png" alt="Linkedin"></a></li>
+                        <li><img src="images/m.svg" alt=""></li>
+                        <li><img src="images/cat.svg" alt=""></li> 
+                        <li><img src="images/hand.svg" alt=""></li>
+                    </ul>
+                </section>
+
+                <!--section 2-->
+                <section class="section-2">
```
Comment (rotimiazeez) [inline index.html:38]: - [ ] This section should have a `display:grid` property according to the project requirement✌

File: index.html [hunk 2]
```diff
@@ -2,32 +2,174 @@
...
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+
+                        <p class="programming_language">
+                        <button>html</button>
+                        <button>css</button>
+                        <button>javaScript</button>
+                        </p> 
+
+                        <button>See Project</button>
```
Comment (rotimiazeez) [inline index.html:59]: - [ ] Please make sure the button follows the button styling: Active, Hover, Pressed and margin ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122))✔

File: index.html [hunk 3]
```diff
@@ -2,32 +2,174 @@
...
+                        <button>javaScript</button>
+                        </p> 
+                        
+                        <button>See Project</button>
+                </section>
+                </section>
+
+                <!--section 6-->
+                <section class="about_my_self">
+                    <h1>About Myself</h1>
```
Comment (rotimiazeez) [inline index.html:143]: - [ ] As stated earlier please make sure you don't have more than one `h1`  tag in your code😊

File: index.html [hunk 4]
```diff
@@ -2,32 +2,174 @@
...
+                    <p class="content">
+                        I’m a software developer! I can help you build a product, feature or website Look through some of my work
+                        and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                    </p>
+                    <a class="connect">LET’S CONNECT</a>
```
Comment (rotimiazeez) [inline index.html:148]: - [ ] Please note that this should use an `hx` tag `h2-h6` to match the project requirement😉

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @ForHemer 👋, 
## Good job so far!👏
### To Highlight🔥
- No Linter errors✔
- Good PR description😉
- Good commit messages👏
- Descriptive README🔥
#### There are some issues that you still need to work on to go to the next project but you are almost there!🚀
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -28,6 +32,143 @@ <h1>I’m Hemerson Glad to see you!</h1>
...
+        </section>
+
+        <!--section 6-->
+        <section class="about_my_self">
+            <h2>About Myself</h2>
+            <p class="content">
+                I’m a software developer! I can help you build a product, feature or website Look through some of my work
+                and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <a class="connect">LET’S CONNECT</a>
```
Comment (petudeveloper) [inline index.html:147]: As mentioned in the previous review this `<a><a/>` tag should be should use an `hx` tag h2-h6 to match the project requirement(If there are other titles (e.g., "Let's connect") use a header tag hx). These embedded semantics offer significant advantages 😉

**Review by petudeveloper — CHANGES_REQUESTED** [review]
# Hi @ForHemer ,  👋  

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Gambit142 — APPROVED** [review]
Hi @ForHemer 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**